magda-web-client/src/Components/Dataset/Add/Pages/DetailsAndContents/index.tsx

```diff
@@ -0,0 +1,350 @@
+                        metadata.
+                    </ToolTip>
+                    <div className="clearfix">
+                        <MultilineTextEditor
+                            value={dataset.description}
+                            placerHolder="Enter description text"
+                            limit={250}
+                            onChange={props.edit("dataset")("description")}
+                        />
+                    </div>
+                </div>
+
+                <h3 className="with-underline">Dates and updates</h3>
+
+                <div className="question-dataset-status">
+                    <h4>What is the status of the this dataset?</h4>
+                    <div className="row">
+                        <div className="col-sm-6">
+                            <ReactSelect
+                                className="react-select"
+                                isMulti={false}
+                                isSearchable={false}
+                                options={Object.keys(
+                                    codelists.datasetCurrencyStatus
+                                ).map(key => ({
+                                    label: codelists.datasetCurrencyStatus[key],
+                                    value: key
+                                }))}
+                                onChange={(item: any) => {
+                                    const status = item.value as CurrentStatusType;
+                                    editCurrency("status")(status);
+                                    if (
+                                        status !== "SUPERSEDED" &&
+                                        currency.supersededBy
+                                    ) {
+                                        editCurrency("supersededBy")([]);
+                                    }
+                                    if (
+                                        status !== "RETIRED" &&
+                                        currency.retireReason
+                                    ) {
+                                        editCurrency("retireReason")("");
+                                    }
+                                }}
+                                styles={ReactSelectStyles}
+                                value={{
+                                    label:
+                                        codelists.datasetCurrencyStatus[
+                                            currency.status
+                                        ],
+                                    value: currency.status
+                                }}
+                            />
+                        </div>
+                    </div>
+                </div>
+
+                {currency.status === "SUPERSEDED" ? (
+                    <div className="question-dataset-superseded-by">
+                        <h4>What dataset has it been superseded by?</h4>
```

Comment:
This should be a plural (i.e. "What dataset(s) has it been superseded by?")

Reply:
Will update~ 👍